zookeeper/init.sls

```diff
@@ -33,15 +33,8 @@ install-zookeeper:
     - group: root
     
 zookeeper-home-link:
-  alternatives.install:
-    - link: {{ zk.alt_home }}
-    - path: {{ zk.real_home }}
-    - onlyif: test -d {{ zk.real_home }} && test ! -L {{ zk.alt_home }}
-    - priority: 30
-    - require:
-      - archive: install-zookeeper
   file.symlink:
     - name: {{ zk.alt_home }}
     - target: {{ zk.real_home }}
```

Comment:
This is how the alternatives system should work in the first place.
I'm not sure why we need those `file.symlink` states, they just only do harm by replacing the correct symlinks which would be normally set by `alternatives`. Perhaps, that was another workaround for some legacy issue.

With the above snippet you'd actually ensure that the `alternative` command has been run correctly...
I think we may just remove the `file.symlink` states and everything should work as it intended to. I see no explicit requirements on them.